modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java

```diff
@@ -775,7 +787,59 @@ public void visit(BLangInvocation iExpr) {
             } else {
                 emit(InstructionCodes.CALL, funcRefCPIndex, funcCallCPIndex);
             }
+        } else if (iExpr.expr != null) {
```

Comment:
You can visit BLangActionInvocation directly in the code generator and generate code.